SmartDeviceLink-iOS/SmartDeviceLink/SDLProtocol.m

```diff
@@ -310,7 +345,7 @@ - (void)sendRawData:(NSData *)data withServiceType:(SDLServiceType)serviceType {
 
 
 #pragma mark - SDLProtocolListener Implementation
-- (void)handleProtocolStartSessionACK:(SDLServiceType)serviceType sessionID:(Byte)sessionID version:(Byte)version {
+- (void)handleProtocolStartSessionACK:(SDLServiceType)serviceType sessionID:(Byte)sessionID hashID:(UInt32)hashID version:(Byte)version {
```

Comment:
Unfortunately, we're both touching the same code here. The security framework deprecates and updates most of these `SDLProtocolListener` calls. This one that you're changing would have necessitated a major version change (so that you are aware in the future). The security framework adds a method that passes the entire SessionACK header, that might require some changes to this on your part.